app/src/main/java/com/sunlightlabs/congress/services/UpcomingBillService.java

```diff
@@ -1,86 +1,61 @@
 package com.sunlightlabs.congress.services;
 
+import com.sunlightlabs.congress.models.Bill;
 import com.sunlightlabs.congress.models.CongressException;
 import com.sunlightlabs.congress.models.UpcomingBill;
 
-import org.apache.http.impl.cookie.DateUtils;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
 import java.text.ParseException;
 import java.util.ArrayList;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.GregorianCalendar;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
 public class UpcomingBillService {
-	
-	private static String[] fields = {
-		"context", "chamber", "legislative_day", "congress", "source_type",
-		"bill_id", "bill", "range"
-	};
-	
-	public static List<UpcomingBill> comingUp() throws CongressException {
-		Map<String,String> params = new HashMap<String,String>();
-		
-		// require an attached bill
-		params.put("bill__exists", "true");
-
-		// soonest first, since this is the future
-		// within a day, list day-specific ones first, then week-specific 
-		params.put("order", "legislative_day__asc,range__asc");
 
-        // only look for upcoming bills whose upcoming date is more than a week ago
-        GregorianCalendar calendar = new GregorianCalendar(DateUtils.GMT);
-        calendar.add(Calendar.DATE, -7);
-        Date now = calendar.getTime();
-        params.put("legislative_day__gte", Congress.formatDateOnly(now));
-		
-		return upcomingBillsFor(Congress.url("upcoming_bills", fields, params, 1, Congress.MAX_PER_PAGE));
```

Comment:
Leaning on server-side defaults, rather than client-specified ordering, for things like the available range, whether bills from today or yesterday are still there, and for putting them in ascending order.